tests/test_playwright.py

```diff
@@ -143,3 +143,38 @@ def check_colors():
                 expect(el).to_have_css("color", hover_color)
 
     _check_test_site(site_name, site_path, check_colors)
+
+
+def test_secondary_sidebar_toc_scrollspy(
+    sphinx_build_factory: Callable,
+    page: Page,
+    url_base: str,
+) -> None:
+    """Test that the secondary sidebar TOC highlights the correct item upon scroll."""
+    site_name = "colors"
```

Comment:
Would you mind creating a new test site in the tests/sites folder, rather than reusing the colors test fixture? It unnecessarily entangles the test for colors with this test.